### PR TITLE
Add documentation about how to install rspec-docs-template yard plugin

### PR DESCRIPTION
**Pull request by benoittgt**
I found I had to do this to properly install this plugin for rspec-rails and other rspec gems rake command.

Related:
- https://github.com/rspec/rspec-rails/pull/2306

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,15 @@ Requires a recent version of Ruby (> 2.1.x, tested on 2.5.3), bundler and imagem
...
 * `middleman build`
 
+## Install rspec-docs-template yard plugin
+
+This plugin is needed when you run Rakefile command to generate documentation from
+`rspec-dev` or `rspec-rails`.
+
+From this repository:
+
+`gem build yard-rspec-docs-template.gemspec`
```
Comment (JonRowe) [inline README.md]: As per my comment in rspec/rspec-rails#2298 I think it's ok to remove this.

Comment (benoittgt) [reply README.md]: So you want to replace this with `ag` command?

Comment (JonRowe) [reply README.md]: Thats in the rake task already

Comment (benoittgt) [reply README.md]: Yes but it needs to be fixed. I will look at it.

**Comment by benoittgt** [comment]
Closing because we no more use the plugin.

https://github.com/rspec/rspec-dev/pull/245

**Pull request closed without merging**